sql/ddl_internal.sql

```diff
@@ -25,3 +25,10 @@ CREATE OR REPLACE FUNCTION _timescaledb_internal.refresh_continuous_aggregate(
     continuous_aggregate     REGCLASS,
     hypertable_chunk         REGCLASS
 ) RETURNS VOID AS '@MODULE_PATHNAME@', 'ts_continuous_agg_refresh_chunk' LANGUAGE C VOLATILE;
+
+-- Drop the specified chunk replica on the specified data node
+CREATE OR REPLACE FUNCTION  _timescaledb_internal.drop_chunk_replica(
+    node_name               NAME,
+    chunk                   REGCLASS
+) RETURNS VOID
```

Comment:
Since the function name is about chunk, then I suggest to put the chunk as the first argument and then the data node name. Other API functions in this file are about data nodes and those the data node is the first argument.

Reply:
Hmm, will have to change lots of files for this. Will do..

Reply:
I agree with Ruslan. We often try to make the main object the first parameter in our API functions. I think it is good with consistence, otherwise it is a slippery slope IMO.

Reply:
Actually, thinking about this some more, maybe the parameter order makes sense if we'd like to expand this to drop multiple chunk replicas in one go:

```
drop_chunk_replicas('dn1', 'chunk1');
drop_chunk_replicas('dn1', 'chunk1', 'chunk2');
```

For the latter case we could make it a vararg function. Or, we could have one `drop_chunk_replica` and one `drop_chunk_replicas` function (obviuosly, the former would just call the latter with one chunk).

Reply:
Have moved `chunk` as the first argument now..

Reply:
That works for now @nikkhils. We fan always change later.